### ISSUE TITLE
sl-app cannot connect to sl-email to send emails

### ISSUE THREAD

**Issue by Spriithy** [simple-login/app#431]
Hey there !
My sl-email container is running with IP 240.0.0.4, is there a way to cold-config the app to use this address instead of the default one ?
Thanks !

EDIT: In your previous reply, you say that postfix is supposed to be accessible at 240.0.0.1 but this actually is the address setup for the gateway no ?

_Originally posted by @Spriithy in https://github.com/simple-login/app/issues/378#issuecomment-814712005_

**Comment by nguyenkims** [comment]
> My sl-email container is running with IP 240.0.0.4, is there a way to cold-config the app to use this address instead of the default one ?

Hey I'm not sure to understand the question ... just curious, why do you want to use this address? 240.0.0.0 is just an arbitrary network choice so it can be whitelisted in Postfix: all docker contains will have 240.0.0.* as IP address and can send emails directly to Postfix without any authentication.

**Comment by Spriithy** [comment]
Hey

Well, I followed the steps in the readme file, but then when I registered, SimpleLogin could not send me the verification email !

So I checked the docker logs of sl-app and saw that the connection to the mailing system timed out..

Then I checked I had correctly setup the network 

```
sl@vps690484:~$ docker exec -it sl-email ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
25: eth0@if26: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default
    link/ether 02:42:f0:00:00:04 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 240.0.0.4/24 brd 240.0.0.255 scope global eth0
       valid_lft forever preferred_lft forever
```

So the emailing app is available at 240.0.0.4, then :


```
sl@vps690484:~$ docker exec -it sl-app telnet 240.0.0.4 25
Trying 240.0.0.4...
telnet: Unable to connect to remote host: Connection refused
```

So I assumed that the sl-app container just didn't know where to find the sl-email server and timed out because of wrong config.

Am I missing anything ?

**Comment by nguyenkims** [comment]
Hey actually for sending emails, the `sl-app` connects to the Postfix run on the host (available at 240.0.0.1) and not to `sl-email`: the sl-email container receives email from Postfix on port 20381. There's no direct connection between sl-app and sl-email.

Can you send an email from your Postfix from your server with this command, run **outside** of docker? Please replace `your_email@example.com` by your actual email. You also might need to install `swaks` which is an email utility program.

```
swaks --to your_email@example.com
```

**Comment by Spriithy** [comment]
It worked, yup. Ended up in spam tho
